fab/services/templates/supervisor_formplayer_spring.conf

```diff
@@ -1,5 +1,5 @@
 [program:{{ project }}-{{ environment }}-formsplayer-spring]
-command=java -jar -Xmx{{ formplayer_memory }} -Xss1024k {{ code_current}}/submodules/formplayer/build/libs/formplayer.jar
+command=java -jar -Xmx{{ formplayer_memory }} -Xss1024k {{ code_current}}/formplayer_build/formplayer.jar
```

Comment:
we don't need to set the location of the config because spring will auto look in the current directory that the formplayer jar is in